client/elements/addons/sc-top-sheet-views.js

```diff
@@ -183,6 +183,14 @@ export class SCTopSheetViews extends LitLocalized(LitElement) {
       label {
         color: var(--sc-on-tertiary-primary-text-color);
       }
+      
+      code.key {
```

Comment:
This is what this looks like in both light and dark themes:

![Screenshot from 2024-08-04 14-29-08](https://github.com/user-attachments/assets/740a87cb-73fe-4d24-a407-5cd7ad942199)

![Screenshot from 2024-08-04 14-29-18](https://github.com/user-attachments/assets/af3f7659-545d-4b56-88ae-4bd852462bad)

Happy to change this styling to conform to whatever style guidelines you have.